portfolio.html

```diff
@@ -20,197 +21,44 @@
-          <div class="cardlistitem">2015</div>
-        </div>
-        <div class="cardabout">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required.
-        </div>
-        <ul class="cardtechnology">
-          <li class="technology">html</li>
-          <li class="technology">css</li>
-          <li class="technology">javaScript</li>
-        </ul>
-        <button type="button" class="btnseeproject">See Project</button>
-      </div>
-      <div class="card">
-        <img
-          class="cardimage"
-          src="./images/Snapshoot_Portfolio_Yoga.png"
-          alt="Yogaimage"
-        />
-        <div class="cardheading">
-          <h2>Tonic</h2>
-        </div>
-        <div class="canopyitemwrap">
-          <div class="cardcanopy">CANOPY</div>
-          <div class="dot cardlistitem">&bull;</div>
-          <div class="cardlistitem">Back End Dev</div>
-          <div class="dot cardlistitem">&bull;</div>
-          <div class="cardlistitem">2015</div>
-        </div>
-        <div class="cardabout">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required.
-        </div>
-        <ul class="cardtechnology">
-          <li class="technology">html</li>
-          <li class="technology">css</li>
-          <li class="technology">javaScript</li>
-        </ul>
-        <button type="button" class="btnseeproject">See Project</button>
-      </div>
-      <div class="card">
-        <img
-          class="cardimage"
-          src="./images/Snapshoot_Portfolio_MultiPostStories.png"
-          alt="multipoststoryimage"
-        />
-        <div class="cardheading">
-          <h2>Multi-Post Stories</h2>
-        </div>
-        <div class="canopyitemwrap">
-          <div class="cardcanopy">CANOPY</div>
-          <div class="dot cardlistitem">&bull;</div>
-          <div class="cardlistitem">Back End Dev</div>
-          <div class="dot cardlistitem">&bull;</div>
-          <div class="cardlistitem">2015</div>
-        </div>
-        <div class="cardabout">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required.
-        </div>
-        <ul class="cardtechnology">
-          <li class="technology">html</li>
-          <li class="technology">css</li>
-          <li class="technology">javaScript</li>
-        </ul>
-        <button type="button" class="btnseeproject">See Project</button>
-      </div>
-    </section>
+    <section class="landingpagecard">
+      <header>
+        <nav class="topnav">
+          <a class="mylogo" href="#mylogo">My Logo</a>
+          <ul class="nav__links">
+            <li class="line"><a class="testing" href="#portfolio">Portfolio</a></li>
+            <li class="line"><a class="testing" href="#about">About</a></li>
+            <li class="line"><a class="testing" href="#contact">Contact</a></li>
+          </ul>
+        </nav>
+      </header>
```

Comment:
## OPTIONAL 

I noticed that your website has two breakpoints and this is what your website looks like it would be better to pick just one breakpoint for your desktop version

Your design at 991px     |  Your design at 992px
:-------------------------:|:-------------------------:
![991](https://user-images.githubusercontent.com/69638013/155749870-91544330-5215-42a2-b74f-b5aff4ef2f82.PNG)  | ![breakpoint](https://user-images.githubusercontent.com/69638013/155749741-c921ef66-5915-4648-9f86-3c6545c44fc3.PNG)

Reply:
I did not understand much about the two breakpoints but tried to implement it.

Reply:
Hi @amrendrakind, kindly check your css files you have two files named portfolio_D.css and portfolio_M.css kindly check the content of these files

Reply:
I updated it. Is it working as per you now?

Reply:
Can you please review and approve my project?

Reply:
> I updated it. Is it working as per you now?

Please note this is optional but it would be better to have just one css style file because both files are duplicate, also the only required change is where you are to make the about and myself section to look like the original design, currently I see that is not effected and I also see that all changes requested in the first and second reviews are now missing due to your recent updates, kindly pick just one css file and one breakpoint for desktop and work with it 👍🏿

Reply:
I am not able to understand what is not working and what  missed. I implemented all changes requested by previous reviewers.

Can you explain me over zoom call so that I really understand it and can work on?

Reply:
> I am not able to understand what is not working and what missed. I implemented all changes requested by previous reviewers.
> 
> Can you explain me over zoom call so that I really understand it and can work on?

- Please send me a message on slack or drop zoom link here

Reply:
Amrendra K is inviting you to a scheduled Zoom meeting.

Topic: My Meeting
Time: Feb 25, 2022 11:30 PM India

Join Zoom Meeting
https://microverse.zoom.us/j/97001375185?pwd=d2NXaHlFa1MzMmZEcnZjdkJNLzUwQT09

Meeting ID: 970 0137 5185
Passcode: 146082
One tap mobile
+19292056099,,97001375185# US (New York)
+12532158782,,97001375185# US (Tacoma)

Dial by your location
        +1 929 205 6099 US (New York)
        +1 253 215 8782 US (Tacoma)
        +1 301 715 8592 US (Washington DC)
        +1 312 626 6799 US (Chicago)
        +1 346 248 7799 US (Houston)
        +1 669 900 6833 US (San Jose)
Meeting ID: 970 0137 5185
Find your local number: https://microverse.zoom.us/u/abreaJIOjl

Reply:
All suggestions/recommendations implemented. It will be great if you can have a look.

Reply:
@prolajumokeoni. Nobody is approving the project despite taken effort to implement all changes. Are you people just to finding issues or you have any responsibility to approve or check if changes implemented?